### ISSUE TITLE
No infinite scrolling in clothes in some components

### ISSUE THREAD

**Issue by vbelevcev** [MAFINS/MenyooSP#544]
No infinite scrolling in clothes in some components mp_f_freemode mp_m_freemode
("Beard/Mask", PV_COMP_BERD); // Beard // FACE_BRD
("Legs", PV_COMP_LOWR);
("Shoes", PV_COMP_FEET); // Shoes // CSHOP_ITEM39
("Accessory/Tops", PV_COMP_ACCS); // Accessory // PIM_TACCE
("Emblem", PV_COMP_DECL);
"Tops2 (Outer)", PV_COMP_JBIB);

Although everything runs smoothly in these components
("Head", PV_COMP_HEAD);
("Hair", PV_COMP_HAIR); // Hair // FACE_HAIR
("Torso", PV_COMP_UPPR);
("Hands/Back", PV_COMP_HAND);
("Teeth/Scarf/Necklace/Bracelets", PV_COMP_TEEF);
("Task/Armour", PV_COMP_TASK);

and since I decided to write about it, who can really fix it, it would not be superfluous to add the entry of his value by number, without scrolling through the list

**Comment by katsutosh** [comment]
For me was the fix, that I removed (from dlclist file) / deactivated the dlc's with _g9ec (next gen) 
The g9ec dlc's includes just placeholder/are empty and because of that, it can't be drawn on pc version.

**Comment by mistahwolf** [comment]
@vbelevcev , yes delete all the 'g9ec' dlc folders and dlclist. As katsutosh said, 'they're just placeholders' for the 'pay-to-play' 'vanillites' on-line. They just cause problems in modded SP. The only one you might want to keep is the one with the 'sentinel' car as it's the only one I believe you can spawn in SP.
